contrib/poly2tri/poly2tri/sweep/sweep.cc

```diff
@@ -231,7 +231,7 @@ void Sweep::FillAdvancingFront(SweepContext& tcx, Node& n)
 
   while (node->next) {
     double angle = HoleAngle(*node);
-    if (angle > M_PI_2 || angle < -M_PI_2) break;
+    if (angle > PI || angle < -PI_2) break;

```

Comment:
PR is ok ofr me :-). And I made more errors like this, beliefe me!

On Thu, Apr 21, 2016 at 12:29 PM, Squareys <notifications@github.com> wrote:

> In contrib/poly2tri/poly2tri/sweep/sweep.cc
> <https://github.com/assimp/assimp/pull/859#discussion_r60558852>:
>
> > @@ -231,7 +231,7 @@ void Sweep::FillAdvancingFront(SweepContext& tcx, Node& n)
> >
> >    while (node->next) {
> >      double angle = HoleAngle(*node);
> > -    if (angle > M_PI_2 || angle < -M_PI_2) break;
> > +    if (angle > PI || angle < -PI_2) break;
>
> Damn, this is embarassing... this kind of mistake in such a simple PR...
>
> @mosra <https://github.com/mosra> Thanks for noticing!
> @kimkulling <https://github.com/kimkulling> Sorry about this. Shall I
> create a PR for the fix or do you want to do the fix yourself?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/assimp/assimp/pull/859/files/7c0e40a3f4126ce336a6b1daa258e3ee1b6a25f5#r60558852>
>